tests/__sf_section_conda.test.fish

```diff
@@ -0,0 +1,80 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+set -l LOCAL_CONDA_VERSION 4.5.11
+
+function setup
+	spacefish_test_setup
+	mock conda 0 "echo \"4.5.11\""
+	mkdir -p /tmp/tmp-spacefish
+	cd /tmp/tmp-spacefish
+end
+
+function teardown
+	rm -rf /tmp/tmp-spacefish
+	if test "$CONDA_DEFAULT_ENV"
+		set -e CONDA_DEFAULT_ENV
+	end
+end
+
+test "Prints section when conda is installed and CONDA_DEFAULT_ENV is set"
+    (
+		set -g CONDA_DEFAULT_ENV some-env
+
+        set_color --bold fff
+		echo -n "via "
+		set_color normal
+		set_color --bold blue
+		echo -n "🅒 v$LOCAL_CONDA_VERSION"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+    ) = (__sf_section_conda)
```

Comment:
Compare this with line 49. Your test endings have inconsistent indentation! :ghost: